bbeam/emails/Review.tsx

```diff
@@ -0,0 +1,168 @@
+    },
+    thumbsDown: {
+      url: "#",
+      imageSrc: assetUrl("/assets/thumbsdown.png"),
+      text: "Not so great",
+    },
+  };
+  return (
+    <MjmlRaw>
+      <tr>
+        <td align="center">
+          <a
+            href={values[value].url}
+            style={{
+              fontFamily: fontFamily.sans,
+              fontSize: fontSize.base,
+              textDecoration: "none",
+              color: colors.black,
+            }}
+          >
+            <table
+              border={0}
+              cellPadding={0}
+              cellSpacing={0}
+              role="presentation"
+              width="100%"
+              style={{
+                tableLayout: "fixed",
+                backgroundColor: colors.blue,
+                borderRadius: borderRadius.base,
+                borderColor: colors.black,
+                borderWidth: "1px",
+                borderStyle: "solid",
+                borderCollapse: "separate",
+                padding: 24,
+              }}
+            >
+              <tr>
+                <td align="center">
+                  <img height={33} width={33} src={values[value].imageSrc} />
+                </td>
+              </tr>
+              <tr>
+                <td align="center">
+                  <p
+                    style={{
+                      fontFamily: fontFamily.sans,
+                      fontSize: fontSize.base,
+                      textDecoration: "none",
+                      textTransform: "uppercase",
+                      color: colors.white,
+                      paddingTop: 16,
+                    }}
+                  >
+                    {values[value].text}
+                  </p>
+                </td>
+              </tr>
+            </table>
+          </a>
+        </td>
+      </tr>
+    </MjmlRaw>
+  );
+};
+
+export default function Review({ email, name }: ReviewProps) {
+  return (
+    <Layout>
+      <Header border />
+
+      <MjmlWrapper fullWidth={true} backgroundColor={colors.white}>
+        <MjmlSection cssClass="gutter" paddingTop={64} paddingBottom={48}>
+          <MjmlColumn>
+            <Heading fontSize={fontSize.xxxl} paddingBottom={24}>
+              {name}, how was lorem ipsum dolor?
+            </Heading>
+            <Text fontSize={fontSize.lg}>
+              Sed ut perspiciatis unde omnis iste natus error sit voluptatem
+              accusantium doloremque laudantium.
+            </Text>
+          </MjmlColumn>
+        </MjmlSection>
+        <MjmlSection cssClass="gutter" paddingBottom={58}>
+          <MjmlGroup>
+            <MjmlColumn paddingRight={"6px"}>
+              {ReviewButton("thumbsUp")}
+            </MjmlColumn>
+            <MjmlColumn paddingLeft={"12px"}>
+              {ReviewButton("thumbsDown")}
+            </MjmlColumn>
+          </MjmlGroup>
+        </MjmlSection>
+        <MjmlSection>
+          <MjmlColumn>
+            <MjmlDivider borderWidth={1} borderColor={colors.black} />
```

Comment:
@elliothursh do you know if there's an easy way to make all of the black dividers in this one full width? if not the header, then how to make the ones in the email body the same width? sorry i feel like i'm missing something obvious here... 

<img width="743" alt="Screen Shot 2022-12-05 at 3 43 54 PM" src="https://user-images.githubusercontent.com/609038/205769486-c1765e18-24d3-48dd-8aad-a3335d8a957b.png">